README.md

```diff
@@ -209,9 +209,208 @@ public void cvurl() {
         System.out.println("CVurl POST: " + result);
     }
 ```
+And another example - due to underlying Mapper (by default it's Jackson) You can send use Java objects as a Request body without a hassle
+```java
+ public UserCreatedResponseDto createUser(UserDto userDto) {
+        return cVurl.POST(HOST + USERS)
+                .body(userDto)
+                .header(HttpHeader.CONTENT_TYPE, MIMEType.APPLICATION_JSON)
+                .build()
+                .asObject(UserCreatedResponseDto.class, HttpStatus.CREATED);
+    }
+``` 
+
 Easy  as pie. 
 Interested? We encourage you to try cVurl!
 
+## Java 11 Http/2 Client missing parts
+
+#### Formdata
+There is no built-in support to send a POST request with x-www-form-urlencoded, 
+and to implement it with plain Java 11 Http/2 client you have to add several lines of code to create BodyPublisher
+```java
+public static BodyPublisher ofFormData(Map<Object, Object> data) {
+    var builder = new StringBuilder();
+    for (Map.Entry<Object, Object> entry : data.entrySet()) {
+      if (builder.length() > 0) {
+        builder.append("&");
+      }
+      builder
+          .append(URLEncoder.encode(entry.getKey().toString(), StandardCharsets.UTF_8));
+      builder.append("=");
+      builder
+          .append(URLEncoder.encode(entry.getValue().toString(), StandardCharsets.UTF_8));
+    }
+    return BodyPublishers.ofString(builder.toString());
+  }
+```
+Well, we bet you would try to avoid writing something like this. Because, that's not all.
+```java
+    var client = HttpClient.newHttpClient();
+
+    Map<Object, Object> data = new HashMap<>();
+    data.put("id", 1);
+    data.put("name", "a name");
+    data.put("ts", System.currentTimeMillis());
+
+    var request = HttpRequest.newBuilder()
+                    .POST(ofFormData(data))
+                    .uri(URI.create("https://localhost:8443/formdata"))
+                    .header("Content-Type", "application/x-www-form-urlencoded")
+                    .build();
+
+    return client.sendAsync(request, BodyHandlers.ofString())
+                    .thenApply(HttpResponse::body)
+                    .exceptionally(e -> "Error: " + e.getMessage())
+                    .thenAccept(System.out::println);
+```
+We do not fancy such verbosity.
+That's why we do all stuff for you under the hood. What You really use is just this simple and nice-looking (fingers crossed) API.
+
+###### cVurl formdata example
+```java
+ public User createUserFromFormUrlencoded(Map<String, String> userMap) {
+        return cVurl.post("https://...")
+                .formData(userMap)
+                .header(HttpHeader.CONTENT_TYPE, MIMEType.APPLICATION_FORM)
```

Comment:
there is no  need to set CONTENT_TYPE header here, formData method does it implicitly.

Reply:
but you do that in examples. Will refactor

Reply:
sorry, my bad, will change examples